src/components/card/thumbnailCard.js

```diff
@@ -20,12 +20,12 @@ export default function ThumbnailCard(props){
 			</span>
 			{/*<span className="thumbCntnr"><img alt="For dummy purpose only" src="https://source.unsplash.com/random/200x200"/></span>*/}
 			{/*<h4>{props.title}</h4>*/}
-			<h4><Link title={props.title} to={props.linkToUrlSlug}>{props.title}</Link></h4>
```

Comment:
The current thumbnail view was a temporary setup and we'd be replacing it altogether once styling for it is finalized!
But, yeas for now, a clickable thumbnail is good to go ✌